hazelcast/src/main/java/com/hazelcast/instance/DefaultOutOfMemoryHandler.java

```diff
@@ -69,8 +70,7 @@ public DefaultOutOfMemoryHandler() {
     }
 
     public DefaultOutOfMemoryHandler(double freeVersusMaxRatio) {
-        this.freeVersusMaxRatio = freeVersusMaxRatio;
-        this.memoryInfoAccessor = new RuntimeMemoryInfoAccessor();
+        this(freeVersusMaxRatio, new RuntimeMemoryInfoAccessor());
```

Comment:
We should use the existing constructor (will help code coverage + reduce errors on setting different fields in multiple constructors)